packages/dds/tree/src/feature-libraries/flex-map-tree/mapTreeNode.ts

```diff
@@ -431,18 +436,23 @@ class EagerMapTreeRequiredField<T extends FlexAllowedTypes>
 	extends EagerMapTreeField<T>
 	implements FlexTreeRequiredField<T>
 {
+	public get editor(): ValueFieldEditBuilder<ExclusiveMapTree> {
+		throw unsupportedUsageError("Setting a required field");
```

Comment:
This seems like a strange error message.

Reply:
`unsupportedUsageError` appends "is not supported for nodes that have not been inserted into the tree" to whatever string is passed in.

Reply:
I guess it's fine that this pops up when fetching the editor, since there's nothing else you can do with that editor.

Reply:
Oh, I see what you mean now. "Editing a required field" kind of makes more sense, but yea, "setting..." is also okay since setting is the only method on the editor. I'm fine with either. It's going to go away soon anyway.